howToTest.md

```diff
@@ -0,0 +1,76 @@
+[Jest](https://jestjs.io/docs/en/expect) main testing framework
+[Enzyme](https://enzymejs.github.io/enzyme) - Helper testing framework for getting elements from the DOM
+
+### Basic settings:
+
+1. Create '**tests**' folder in component's folder
+2. Create <my-component>.spec.tsx
+3. First line must be (it's IMPORTANT) **import 'jsdom-global/register';**
+4. Then import :
+   - import React from 'react';
+   - import { MyCompnent } from '../MyCompnent'; // your components
+
+### Unit testing of component:
+
+1.  import { shallow } from 'enzyme';
```

Comment:
Please put every piece of code inside specified code-block:  \`\`\`javascript \`\`\`